learning/katas/go/core_transforms/windowing/additional_parameters/task.md

```diff
@@ -0,0 +1,77 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+# Windowing
+
+This lesson introduces the concept of windowed PCollection elements.  A window is a view into a fixed beginning and 
+fixed end to a set of data.  In the beam model, windowing subdivides a PCollection according to the
+timestamps of its individual elements.
+
+Beam can pass information about the window and timestamp to your elements in your DoFn.  All your previous
+lessons' DoFn had this information available, yet you never made use of it in your DoFn parameters.  In this 
+lesson you will.  The simple toy dataset has five git commit messages and their timestamps 
```

Comment:
I'd remove either `simple` or `toy`, they're redundant together.

Reply:
I like the idea of more windowing lessons after the rest of the originally planned katas are complete.